pkg/lockfile/parse-npm-lock-v1_test.go

```diff
@@ -415,3 +415,36 @@ func TestParseNpmLock_v1_OptionalPackage(t *testing.T) {
 		},
 	})
 }
+
+func TestParseNpmLock_v1_SamePackageDifferentGroups(t *testing.T) {
+	t.Parallel()
+
+	packages, err := lockfile.ParseNpmLock("fixtures/npm/same-package-different-groups.v1.json")
+
+	if err != nil {
+		t.Errorf("Got unexpected error: %v", err)
+	}
+
+	expectPackages(t, packages, []lockfile.PackageDetails{
+		{
+			Name:      "eslint",
+			Version:   "1.2.3",
+			Ecosystem: lockfile.NpmEcosystem,
+			CompareAs: lockfile.NpmEcosystem,
+			DepGroups: []string{"dev"},
+		},
+		{
+			Name:      "table",
+			Version:   "1.0.0",
+			Ecosystem: lockfile.NpmEcosystem,
+			CompareAs: lockfile.NpmEcosystem,
+			DepGroups: nil,
```

Comment:
```suggestion
```

Reply:
hmm nope still doesn't work

Reply:
It might just be cause it's on mobile, but are you actually suggesting a multiline edit?

I believe suggestions can't delete the whole selection - they've got to modify, so for deletes you have to suggest modifying the previous and next line as part of the suggestion